index.html

```diff
@@ -248,5 +248,51 @@ <h3 class="about-skills">skills</h3>
         </div>
       </div>
     </section>
+
+    <section class="form-section">
```

Comment:
As mentioned by the previous reviewer, kindly I suggest you stick to the design as much as possible.

Please follow the design.

> ## The Figma Design
>
><img width="203" alt="Screenshot 2023-06-08 at 9 08 34 PM" src="https://github.com/DHE19/portfolio-microverse/assets/91237525/f6b391d6-e98d-4ec1-89d1-f4e8d7b87da7">
>
> ## Your Design
>
><img width="373" alt="Screenshot 2023-06-08 at 9 08 47 PM" src="https://github.com/DHE19/portfolio-microverse/assets/91237525/39292516-b35e-460e-862f-18496229e4ca">

Reply:
Hello @stanleySimeon, thanks 

Could you tell me what's the width of your device? Because for example for  iPhone XR, iPhone 12, Samsung 8, and Samsung s20, and pixel 5  it's working well:


**Iphone XR**
![XR](https://github.com/DHE19/portfolio-microverse/assets/5220260/ef28a15c-9b12-4b96-90dd-7c8b6006b120)

**Iphone 12 Pro**
![Iphone pro 12](https://github.com/DHE19/portfolio-microverse/assets/5220260/4d091542-be80-498a-9811-533996ccd2ed)

**Pixel 5**
![Pixel 5](https://github.com/DHE19/portfolio-microverse/assets/5220260/a47d4ae9-28a1-46a8-8ef5-708658a58211)

Reply:
Also applying the minimum query for mobile devices is working well:

![media](https://github.com/DHE19/portfolio-microverse/assets/5220260/e33eccbc-f706-4006-ad48-fae5ab162d62)